packages/elements/src/components/ino-carousel/ino-carousel.tsx

```diff
@@ -28,13 +34,16 @@ export class InoCarousel implements ComponentInterface {
    * Optional group value to manually manage the displayed slide
    */
   @Prop() value?: string;
-
   @Watch('value')
   valueChanged(newVal: string) {
     this.addAnimationToSlide(this.slides[this.currentSlide]); // adds the slide animation to the current slide
     this.selectSlide(newVal);
     this.addAnimationToSlide(this.slides[this.currentSlide]); // adds the slide animation to the new slide
   }
+  /**
+   * Emits the value of the slide once it changed
```

Comment:
The slides have not been changed on arrow press. We inform the consumer that the slide should be changed.
```suggestion
   * Emits the `value` of the slide that should be displayed after the left or right arrow has been clicked.
```